### ISSUE TITLE
Check if pull request #53 requires changes in the LLR plotting tools.

### ISSUE THREAD

**Issue by sboeser** [sboeser/pisa#54]
In PR #53 the ouput format of the LLR analysis is changed/fixed from 
```
output[data_tag]['false_h_best_fit'] = trials
```
to
```
output[data_tag]['false_h_best_fit']['false_h_settings'] = false_h_settings
output[data_tag]['false_h_best_fit']['llh_null'] = llh_data
output[data_tag]['false_h_best_fit']['trials'] = trials
```
Before, the `false_h_settings` and `llh_null` had been overwritten by the `trials`. Needs checking if this requires changes in the plotting scripts for the LLR results.

**Comment by thehrh** [comment]
The trials combined into a single hdf file serve as the input for the plotting script. I've changed the processing script accordingly, so that the output (the structure of the hdf file) is unchanged despite the additional 'trials' entry above (ba09570c8b25626949795bc74ad8bc8bebf717cf).

**Comment by sboeser** [comment]
Does this mean we can close this issue?

**Comment by thehrh** [comment]
Not quite, since the fix isn't part of the master branch yet. It's only in one of my personal branches, which I'm still developing to a certain extent: https://github.com/sboeser/pisa/compare/master...thehrh:mod_llr_processing. I could already create a PR, however, and let you know as soon as I see it fit to be merged.
